readme.md

```diff
@@ -97,6 +97,36 @@ config.set('foo', '1');
 
 **Note:** The `default` value will be overwritten by the `defaults` option if set.
 
+### migrations
+
+Type: `Object`
+
+You can use migrations to perform operations to the store whenever a version is switched.
+
+The `migrations` object should be consisted of a key-value pair of `version`: `handler`.
+
+**Note**: The `projectVersion` option should be specified in order for the migrations to be run.
+But you only need to specify the version if you don't have a package.json file in your project.
```

Comment:
Perhaps change this line to emphasize that we assume the value of the `version` key. The `version` field might not be set in the package.json.

> You only need to specify the `projectVersion` if there's no version defined in the project's package.json file or there's no package.json at all.

Reply:
Also, I think it's worth documenting the `projectVersion` option, after the `projectName`  one.

Reply:
Agree! Nice catch.